compare_compilers.sh

```diff
@@ -52,7 +50,7 @@ function remove_line()
 {
     local file="$1"
     local line="$2"
-    echo "  Deleting $line from $file"
+    echo -e "  ${GREEN}Deleting $line from $file${RESET}"
```

Comment:
For consistency with the previous echo

```suggestion
    echo -e "  ${YELLOW}Deleting $line from $file${RESET}"
```